tests/testIntegration.nim

```diff
@@ -9,6 +9,7 @@ import ./ethertest
 import ./contracts/time
 import ./integration/nodes
 import ./integration/tokens
+import ./integration/testblockexpiration
```

Comment:
Maybe it's time split this out into it's own module? One for contract negotiation and another for block maintenance? We usually have the `testSomething.nim` reserved for calling the individual test units.

Reply:
Don't have to be in this PR btw...